FogCreek/Glitch-Community
# Whispering handball: Add My Stuff Button to featured projects, edited Notes placement

## Links
* Remix link: https://whispering-handball.glitch.me
* Issue-tracker link: https://github.com/FogCreek/Glitch-Community-Work/issues/302

## GIF/Screenshots:
Add to my stuff button on featured embeds:
![atms-featured](https://user-images.githubusercontent.com/519336/62738891-6abcc300-ba01-11e9-8328-5a7387ddfe63.gif)

Should be responsive:
![image](https://user-images.githubusercontent.com/519336/62739090-fcc4cb80-ba01-11e9-9c3c-7c52663e2b62.png)



Edited placement of Notes to accommodate My Stuff button:
![image](https://user-images.githubusercontent.com/519336/62738942-92ac2680-ba01-11e9-8324-56da707864db.png)


## Changes:
* Added My Stuff button to featured embeds
* Edited My Stuff Button and notes positioning to prevent overlap
* Edited My Stuff Button margins / padding to remove unnecessary padding around My Stuff Button (reduce hit zone)

## How To Test:
* Enable My Stuff at https://whispering-handball.glitch.me/secret
* Log into whispering-handball and go to one of your collections
* Feature a project in your collection and make sure you can add it to the My Stuff collection

## Feedback I'm looking for:
Did I break anything?
